404.html

```diff
@@ -83,11 +83,17 @@ <h3>DietPi Docs</h3>
 			</svg>
 			<h3>DietPi Forum</h3>
 		</a>
-		<a href="https://github.com/MichaIng/DietPi">
+		<a href="https://dietpi.com/blog/">
+            <svg viewBox="-200 -200 1400 1400" aria-hidden="true">
+                <path d="M983.9 420.4c6.1-12.2 6.1-30.6 0-42.9L941 334.6c-6.1-6.1-12.2-6.1-24.5-6.1-6.1 0-18.4 0-24.5 6.1L561.3 659.2v24.5V745h98l324.6-324.6zM193.8 193.7h612.5V255H193.8v-61.3h0zm0 183.8h569.6l42.9-42.9v-18.4H193.8v61.3zm0 306.2H500V745H193.8v-61.3h0zm0-183.7h447.1l61.2-61.3H193.8V500zm734.9 61.2v245c0 67.4-55.1 122.5-122.5 122.5H193.8c-67.4 0-122.5-55.1-122.5-122.5V193.7c0-67.4 55.1-122.5 122.5-122.5h612.5c67.4 0 122.5 55.1 122.5 122.5v73.5c18.4 0 36.8 12.2 55.1 24.5l6.1 6.1v-104C990 89.6 910.4 10 806.3 10H193.8C89.6 10 10 89.6 10 193.8v612.5C10 910.4 89.6 990 193.8 990h612.5C910.4 990 990 910.4 990 806.3V500l-61.3 61.2zm-734.9 61.3h324.6l6.1-6.1 55.1-55.1H193.8v61.2z"/>
+			</svg>
+			<h3>DietPi Blog</h3>
+		</a>
+        <a href="https://github.com/MichaIng/DietPi">
```

Comment:
Take care with tabs vs space indentation:
```suggestion
			<svg viewBox="-200 -200 1400 1400" aria-hidden="true">
				<path d="M983.9 420.4c6.1-12.2 6.1-30.6 0-42.9L941 334.6c-6.1-6.1-12.2-6.1-24.5-6.1-6.1 0-18.4 0-24.5 6.1L561.3 659.2v24.5V745h98l324.6-324.6zM193.8 193.7h612.5V255H193.8v-61.3h0zm0 183.8h569.6l42.9-42.9v-18.4H193.8v61.3zm0 306.2H500V745H193.8v-61.3h0zm0-183.7h447.1l61.2-61.3H193.8V500zm734.9 61.2v245c0 67.4-55.1 122.5-122.5 122.5H193.8c-67.4 0-122.5-55.1-122.5-122.5V193.7c0-67.4 55.1-122.5 122.5-122.5h612.5c67.4 0 122.5 55.1 122.5 122.5v73.5c18.4 0 36.8 12.2 55.1 24.5l6.1 6.1v-104C990 89.6 910.4 10 806.3 10H193.8C89.6 10 10 89.6 10 193.8v612.5C10 910.4 89.6 990 193.8 990h612.5C910.4 990 990 910.4 990 806.3V500l-61.3 61.2zm-734.9 61.3h324.6l6.1-6.1 55.1-55.1H193.8v61.2z"/>
			</svg>
			<h3>DietPi Blog</h3>
		</a>
		<a href="https://github.com/MichaIng/DietPi">
```